components/dashboard/src/settings/SettingsPage.tsx

```diff
@@ -15,7 +15,7 @@ export interface Props {
 }
 
 export function SettingsPage(p: Props) {
-    return <div className="max-w-6xl">
+    return <div className="w-full">
```

Comment:
**issue**: This also affected other settings pages that don't need to span across the right side, see `/account`. Could we bring back this padding for pages like `/account`? Happy to discuss this in a follow up issue. /cc @AlexTugarev

Reply:
All right. I'll revert.

Reply:
**note**: Integrations page should use the full width as is. Only accounts and other pages without much information should be limited to the middle. /cc @AlexTugarev